scalding-parquet-scrooge/src/main/scala/com/twitter/scalding/parquet/scrooge/PartitionedParquetScroogeSource.scala

```diff
@@ -0,0 +1,46 @@
+package com.twitter.scalding.parquet.scrooge
+
+import _root_.cascading.scheme.Scheme
+import _root_.cascading.tuple.Fields
+import com.twitter.scalding._
+import com.twitter.scalding.parquet.thrift.ParquetThriftBase
+import com.twitter.scalding.typed.{ PartitionSchemed, PartitionUtil }
+import com.twitter.scrooge.ThriftStruct
+
+/**
+ * Scalding source to read or write partitioned Parquet scrooge data.
+ *
+ * For writing it expects a pair of `(P, T)`, where `P` is the data used for partitioning and
+ * `T` is the scrooge object. Below is an example.
+ * {{{
+ * val data = MyScroogeObject()
+ * IterablePipe(data, flowDef, mode)
+ *  .write(PartitionedParquetScroogeSource[(String, String), MyScroogeObject](path, "%s/%s"))
```

Comment:
This isn't quite right is it? The partition key needs to be exposed right?
how about:

```
val data: TypedPipe[MyScroogeObject] = ???
data.map { obj => 
  ((obj.country, obj.city), obj) 
}.write(PartitionedParquetScroogeSource[(String, String), MyScroogeObject](path, "%s/%s")
```